lib/socket/listener.js

```diff
@@ -2,11 +2,10 @@
 
 var _ = require('lodash');
 var async = require('async');
-var config = require('config');
 
 var fetcher = require('../github-fetcher');
 
-var pollTimeout = config.get('github.poll-timeout') * 1000;
+var pollTimeout = process.env.GITHUB_POLL_TIMEOUT || 60 * 1000;
```

Comment:
I got tricked by this line when  tried to deploy the application.

Since the timeout configuration was previously set in seconds, so when setting the environment variables to 60 here, the actual polling time is 60ms.

You should wrap the instanciation between parenthesis!